src/components/wizard-upload-configfile/wizard-upload-configfile.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-md-6">
```

Comment:
we should move this into the component that uses this one (design stuff is in two components otherwise)